README.md

```diff
@@ -1 +1,24 @@
-# Hello
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
```

Comment:
- [ ] README file should have a title and description about the project even if it's one or two-line so that anyone can understand your project. Also, include the steps on how to clone and run the project locally. Currently, you are using the default README which rails generate. Please update that. You can follow [this template](https://github.com/microverseinc/readme-template) as well.